lnrpc/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM golang:1.15.7-buster
+FROM lnd-go-base:latest
 
-RUN apt-get update && apt-get install -y \
+RUN apk add --no-cache --update alpine-sdk \
   git \
-  protobuf-compiler='3.6.1*' \
-  clang-format='1:7.0*'
+  bash \
+  protoc=3.13.0-r2 \
+  clang=10.0.1-r0
```

Comment:
Nice! I'm very surprised this doesn't result in a diff when running `make rpc` for this commit. So we get updated versions of `clang` and `protoc` for free :tada:

Reply:
yeah i was surprised by that as well 😃